wordle-web/components/GameBoardGuess.vue

```diff
@@ -1,18 +1,52 @@
 <template>
-  <div>
-    <LetterResult
-      v-for="(letter, k) of guess.letters"
-      :key="k"
-      :letter="letter"
-    />
-  </div>
+  <v-row dense class="justify-center">
+    <v-col :class="isShaking ? 'shaking' : ''" cols="auto" v-for="(letter, k) of guess.letters" :key="k">
+      <LetterResult :letter="letter" />
+    </v-col>
+  </v-row>
 </template>
 
 <script setup lang="ts">
 import { Word } from "~/scripts/word";
 import { defineProps } from "vue";
 
-defineProps<{
+const props = defineProps<{
   guess: Word;
 }>();
+
+const isShaking = ref(false);
+
+watch(() => props.guess.isFilled,
+() => {
+  if(!props.guess.isFilled){
+    isShaking.value = true;
+    setTimeout(() => {
+      isShaking.value = false;
+    }, 300);
+  }
+});
```

Comment:
Watch has two implicit argument's in the callback it produces, accessed by providing variable names to the callback (second parameter of the watch method). The first parameter is the new value being received by the watched property, the second parameter is the old value.

You can access this new value by just supplying one variable, and use that in your if check instead of getting props.guess.isFilled again.

https://vuejs.org/guide/essentials/watchers.html

```suggestion
watch(() => props.guess.isFilled,
(value) => {
  if(value){
    isShaking.value = true;
    setTimeout(() => {
      isShaking.value = false;
    }, 300);
  }
});
```